renpy/display/behavior.py

```diff
@@ -222,8 +222,7 @@ def map_event(ev, keysym):
 
         return False
 
-    if isinstance(keysym, list):
-        keysym = tuple(keysym)
+    keysym = renpy.easy.to_tuple(keysym)
```

Comment:
There doesn't appear to be any expectation here that the `keysym` argument wouldn't already be an `Iterable` of some kind, and I don't think we should be accidentally adding more expensive code here to allow it. These are internal functions that already work, in contrast to the proposed changes to `layeredimage` and `at_list` code which is dealing directly with creator input and providing a direct benefit.

Reply:
I was under the impression that the keysym directly came from the user, even though the function itself is not public ? I don't know, but if non-list iterables of keysyms are currently accepted as well as lists, then we can surely kick it off.

Reply:
I didn't get the rationale quite right, but did, I believe, get the right conclusion. The correct explanation is that it comes down to how `compile_event` works, and that converting non-iterables into iterables here needlessly creates more work in that function, and results in more complex dynamic code being generated there.